src/UI/Art/SpriteArtist/OverlayArtist.java

```diff
@@ -0,0 +1,110 @@
+package UI.Art.SpriteArtist;
+
+import java.awt.AlphaComposite;
+import java.awt.Color;
+import java.awt.Composite;
+import java.awt.Graphics;
+import java.awt.Graphics2D;
+import java.awt.image.BufferedImage;
+import java.util.ArrayList;
+import java.util.Collection;
+
+import Engine.XYCoord;
+import Terrain.GameMap;
+import UI.GameOverlay;
+
+public class OverlayArtist
+{
+  private static final Color HIGHLIGHT_COLOR = new Color(255, 255, 255, 160); // white
+
+  public static final Color MOBILE_FIRE_EDGE = new Color(240,   0,   0, 255); // red
+  public static final Color SIEGE_FIRE_EDGE  = new Color(  0,   0, 160, 255); // dark blue
+  public static final Color FIRE_FILL        = new Color(  0,   0,   0,   0);
```

Comment:
Huh.

Reply:
There wasn't a "transparent" option that I could find.

...which isn't to say there *isn't* a transparent option...

Reply:
I mean. You found it. But why have it if it's transparent? Future plans?

Reply:
It's required to have a "fill" color in an overlay.
I guess I could allow it to be null, and it would save processor cycles in the end?